config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -166,7 +171,7 @@ presubmits:
     extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: main
+      base_ref: release-1.8
```

Comment:
We need to get https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3388 into release-1.8

Reply:
added a hold so we can get that in, feel free to remove the hold when ready